src/pages/Ahnsangjun/CommentList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CommentItem from './CommentItem';
+
+function CommentList(props) {
+  return (
+    <div>
+      {props.todoList.map(item => (
+        <CommentItem item={item} />
+      ))}
+      <CommentItem />
+    </div>
```

Comment:
불필요하게 감싸고 있는 div 태그. React.Fragment로 대체 가능
```suggestion
    <>
      {props.todoList.map(item => (
        <CommentItem item={item} />
      ))}
      <CommentItem />
    </>
```